tests/_support/AcceptanceTester.php

```diff
@@ -12,15 +12,216 @@
+	{
+		$I = $this;
+		$I->amOnPage('administrator/index.php?option=com_content&view=articles');
+		$I->fillField('#filter_search', $title);
+		$I->click('.icon-search');
+		$I->checkAllResults();
+	}
+
+	/**
+	 * @When I feature the content with title :arg1
+	 */
+	public function iFeatureTheContentWithTitle($title)
+	{
+		$I = $this;
+		$I->click(['xpath' => "//div[@id='toolbar-featured']/button"]);
+	}
+
+	/**
+	 * @Then I save and see the :arg1 message
+	 */
+	public function iSaveAndSeeTheMessage($arg1)
+	{
+		$I = $this;
+		$I->waitForText('Articles', 60, ['css' => '.page-title']);
+		$I->see('1 article featured.', ['id' => 'system-message-container']);
+	}
+
+	/**
+	 * @Given I select the content article with title :arg1
+	 */
+	public function iSelectTheContentArticleWithTitle($title)
+	{
+		$I = $this;
+		$I->amOnPage('administrator/index.php?option=com_content&view=articles');
+		$I->fillField('#filter_search', $title);
+		$I->click('.icon-search');
+		$I->checkAllResults();
+		$I->click(['xpath' => "//div[@id='toolbar-edit']/button"]);
+	}
+
+	/**
+	 * @Given I set access level as a register
+	 */
+	public function iSetAccessLevelAsARegister()
+	{
+		$I = $this;
+		$I->selectOptionInChosenById('jform_access', 'Registered');
+
+	}
+
+	/**
+	 * @When I save the article :arg1
+	 */
+	public function iSaveTheArticle($arg1)
+	{
+		$I = $this;
+		$I->clickToolbarButton('Save & Close');
+	}
+
+	/**
+	 * @Given I have article with name :arg1
+	 */
+	public function iHaveArticleWithName($title)
+	{
+		$I = $this;
+		$I->amOnPage('administrator/index.php?option=com_content&view=articles');
+		$I->fillField('#filter_search', $title);
+		$I->click('.icon-search');
+		$I->checkAllResults();
+	}
+
+	/**
+	 * @When I unpublish :arg1
+	 */
+	public function iUnpublish($title)
+	{
+		$I = $this;
+		$I->click(['xpath' => "//div[@id='toolbar-unpublish']/button"]);
+
+	}
+	/**
+	 * @Then I see article unpublish message :arg1
+	 */
+	public function iSeeArticleUnpublishMessage($arg1)
+	{
+		$I = $this;
+		$I->waitForText('Articles', 60, ['css' => '.page-title']);
+		$I->see('1 article unpublished.', ['id' => 'system-message-container']);
+	}
+
+
+	/**
+	 * @Given I have :arg1 content article which needs to be Trash
+	 */
+	public function iHaveContentArticleWhichNeedsToBeTrash($title)
+	{
+		$I = $this;
+		$I->amOnPage('administrator/index.php?option=com_content&view=articles');
+		$I->fillField('#filter_search', $title);
+		$I->click('.icon-search');
+		$I->checkAllResults();
+	}
+
+	/**
+	 * @When I Trash the article with name :arg1
+	 */
+	public function iTrashTheArticleWithName($title)
+	{
+		$I = $this;
+		$I->click(['xpath' => "//div[@id='toolbar-trash']/button"]);
+	}
+	/**
+	 * @Then I see article trash message :arg1
+	 */
+	public function iSeeArticleTrashMessage($arg1)
+	{
+		$I = $this;
+		$I->waitForText('Articles', 60, ['css' => '.page-title']);
+		$I->see('1 article trashed.', ['id' => 'system-message-container']);
+	}
```

Comment:
This method `iSeeArticleTrashMessage` is similar to `iSeeArticleUnpublishMessage`:

```php
/**
 * @Then I see article unpublish message :arg1
 */
public function iSeeArticleUnpublishMessage($arg1)
```

No need to be done now, but if you identify common patterns, maybe we can reconsider them for refactor and make them reusable:

```php
public function iSeeMessage($message)
{
    $I = $this;
    $I->waitForElement(['id' => 'system-message-container'], 60);
    $I->see($message, ['id' => 'system-message-container']);
}